files/zh-cn/web/javascript/reference/statements/with/index.md

```diff
@@ -45,7 +45,7 @@ function f(x, o) {
 
 `f`被调用时，`x`有可能能取到值，也可能是`undefined`，如果能取到，有可能是在 o 上取的值，也可能是函数的第一个参数`x`的值（如果 o 中没有这个属性的话）。如果你忘记在作为第二个参数的对象 o 中定义`x`这个属性，程序并不会报错，只是取到另一个值而已。
 
-**弊端：**使用`with`语句的代码，无法向前兼容，特別是在使用一些原生数据类型的时候。看下面的例子：
+**弊端：** 使用`with`语句的代码，无法向前兼容，特別是在使用一些原生数据类型的时候。看下面的例子：
```

Comment:
Hi @Lzzzs, please follow the [guidelines](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E4%B8%AD%E6%96%87%E5%92%8C%E6%8B%89%E4%B8%81%E8%AF%AD%E7%B3%BB%E6%96%87%E5%AD%97%E9%97%B4%E5%8A%A0%E7%A9%BA%E6%A0%BC). It's better to move `**` beforehead

```suggestion
**弊端**：使用 `with` 语句的代码，无法向前兼容，特別是在使用一些原生数据类型的时候。看下面的例子：
```

Reply:
okey